src/components/sidebar/custom/CustomDialog.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
This shouldn't be called a dialog since it just resides in a tab now. Maybe call it `CustomEventForm` or something.